content/docs/overview/some_terms.md

```diff
@@ -36,8 +36,37 @@ If you right away want to deep-dive into the topic, you can have a look at the [
 
 ## What are Artifacts?
 
-- `Sources`: input for creating resources e.g. source code
-- `Resources`: contain the code to “do” something, e.g. OCI Images, binaries, etc.
-- Have a component-local identity (name and version)
-- Have a `Type` (similar to a MIME-Type)
-- Have an exchangeable `Access` (formal description how to retrieve an artifact, e.g. a download URL)
+- Can be either of kind
+  - `sources`: input for creating resources e.g. source code or
+  - `resources`: contain the code to “do” something, e.g. OCI images, binaries, etc.
```

Comment:
>  contain the code to “do” something,
I do not think this is a great choice of words. If we want to keep this more abstract, why not use the term "payload" instead?